routes/user.js

```diff
@@ -11,37 +11,19 @@ var crypto = require('crypto')
 var nodemailer = require('nodemailer')
 var config = require('config')
 
-// router.get('/profile', function(req, res, next){
-//   res.render('profile', {});
-// })
+//nodemailer obj
+var smtpTransport = nodemailer.createTransport('SMTP', {
+    service: 'SendGrid',
+    auth: {
+        user: process.env.SENDGRID_USER || config.get('emailVerification.user'),
+        pass: process.env.SENDGRID_PASS || config.get('emailVerification.pass')
+    }
+})
 
-// router.get('/library/search', function(req, res, next) {
-//     var tag = req.query.q
-//     var messages = req.flash()
-//     User.findOne({
-//             _id: req.user._id
-//         })
-//         .populate('library')
-//         .exec(function(err, user) {
-//             if (err) return next(err)
-//             Playlist.find({
-//                 owner: user._id
-//             }, function(err, playlists) {
-//                 if (err) return next(err)
-//                 res.render('library', {
-//                     songs: user.library.filter((s) => {
-//                         return (s.title.search(tag) !== -1 ||
-//                             s.lang.search(tag) !== -1 ||
-//                             s.author.search(tag) !== -1)
-//                     }),
-//                     playlists: playlists,
-//                     messages: messages
-//                 })
-//             })
-//         })
-// })
 
+//this route is to handle /library
```

Comment:
Instead of just saying it handles /library, explain what the library is and shows

The library route will provide the data to show the user the songs they have added to their library.
These are songs they may want to use in the future, but haven't necessarily put into a playlist for use.
It gives them a smaller view into the database of songs, of just the songs they are interested in instead of everything.